extensions/wikia/Wall/WallHistoryController.class.php

```diff
@@ -7,7 +7,7 @@ public function __construct() {
 		parent::__construct();
 	}
 
-	public function index() {
+	public function index( $wallMessagesPerPage = null ) {
```

Comment:
This was producing a warning about not matching the signature of the parent method which vomited a stack trace on my dev box.  This brings it into alignment, though the parameter is unused here.